bezier_library/src/bezier_library.cpp

```diff
@@ -299,8 +309,7 @@ bool Bezier::defectIntersectionOptimisation(vtkSmartPointer<vtkPolyData> &poly_d
       double closestPoint[3];
       defectPolyData_->GetPoint(iD, closestPoint);
       // Get direction vector
-      Eigen::Vector3d direction_vector = Eigen::Vector3d(closestPoint[0] - pt[0],
-                                                         closestPoint[1] - pt[1],
+      Eigen::Vector3d direction_vector = Eigen::Vector3d(closestPoint[0] - pt[0], closestPoint[1] - pt[1],
```

Comment:
Why did you put only the first and the second componant and the same line and not the third one?